man/mkosi.1

```diff
@@ -2040,7 +2044,7 @@ Hostname=image
 .SH REQUIREMENTS
 .PP
 mkosi is packaged for various distributions: Debian, Ubuntu, Arch (in
-AUR), Fedora, OpenMandriva.
+AUR), Fedora, OpenMandriva, Gentoo.
```

Comment:
As I learnt myself a few days ago, this file is generated from `mkosi.md`. So those changes should be added there instead.